docs/use-cases/wallet.md

```diff
@@ -0,0 +1,140 @@
+
+*Implementation details:*
+
+The information about the token and its price currently have to be retrieved from different sources (Ethereum node, such as Infura, and one of the public price feeds, such as ...), which makes it more difficult to display the full token details and the balance in different fiat currencies.
+
+#### 2.1.2 Make Deposits
+
+It is required to have funds deposited into OMG Network first before making any transfer. You can implement this functionality by following [Deposit Funds guide](https://docs.omg.network/network/deposits).
+
+*Implementation details:*
+
+Each deposit has a [deposit finality period](https://docs.omg.network/glossary#deposit-finality-period) before it can be used on the network. Currently this period equals 10 Ethereum blocks. The confirmations are counted on the Ethereum because the OMG Network relies on rootchain (Ethereum) security and creates blocks only when new transactions are being formed (i.e. on-demand). You may use the `waitForChildchainBalance` function from [omg-js documentation](https://docs.omg.network/omg-js/) or implement your custom helper/service for polling the expected amount after the deposit.
+
+#### 2.1.3 Make Transfers
+
+Making transfers on the OMG Network has the same behavior as on other networks, blockchains, or payment systems. Currently, the network supports only payment transactions, more advanced types will be available in the future. This operation will occupy the majority of data load, thus should be treated with more focus. You can implement a standard transfer functionality using [Make a Transfer guide](https://docs.omg.network/network/transfers).
+
+*Implementation details:*
+
+When you send a transaction, you will often use [`account/get_utxos`](https://docs.omg.network/elixir-omg/docs-ui/?urls.primaryName=master%2Finfo_api_specs#/Account/account_get_utxos) endpoint to get information about the latest UTXOs. Current implementation returns only confirmed UTXOs that are not used for the pending transaction. One of the ways to mitigate this limitation is to block/diable your wallet UI to prevent sending other transactions until the current transaction is confirmed. Otherwise, you may encounter `UTXO not found` issue. Alternatively, you can keep track of all of the UTXOs used for your transactions but this will create additional complexity for your application.
+
+![](/img/wallet/01.png)
+
+#### 2.1.4 Make Withdrawals (Exits)
+
+Using OMG Network as a value transfer layer offers a faster and cheaper alternative if your work requires frequent operations on the Ethereum network. Most of the time you will hold a certain amount of assets on the OMG Network, and periodically withdraw your funds back to Ethereum. This is the most complicated technical and UX flow. You can implement exits functionality using [Deposit Funds guide](https://docs.omg.network/network/deposits).
```

Comment:
the link to implement exits functionality is linked to the wrong guide (deposit)